documentation/modules/cruise-control/proc-cruise-control-moving-data.adoc

```diff
@@ -0,0 +1,339 @@
+[id='proc-cruise-control-moving-data-{context}']
+= Using Cruise Control to reassign partitions on JBOD disks
+
+[role="_abstract"]
+If you are using JBOD storage and have Cruise Control installed with Strimzi, you can move reassign partitions and move data between the JBOD disks used for storage on the same broker.
```

Comment:
"you can move reassign partitions" ... did you mean just "you can reassign ..." ?